notebooks/gauss_plot.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        latexify(n_figures=2, fig_height=1.5)
</pre><p>Syntax is changed now. Check docstring to know more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>